### DISCUSSION TITLE
PSI is not showing any results for Shopify

### DISCUSSION THREAD

**Discussion by civilixed1996** [GoogleChrome/lighthouse#13401] [Q&A]
This is after new update. How long will it take to solve this issue?
Kindly give some estimate.

**Comment by connorjclark** [comment]
#13379 

No estimate, but you can subscribe to that issue to be alerted of the fix.

**Reply by connorjclark** [reply]
Of course, you already knew that. I can't give an estimate yet but am actively working on the fix.
